soccer/Robot.hpp

```diff
@@ -244,14 +244,29 @@ class OurRobot: public Robot {
 
 
 	/**
-	 * enable kick when ready at a given strength
+	 * enable kick when ready at a given strength (0.0 - 1.0)
 	 */
-	void kick(uint8_t strength);
+	void kick(float strength);
 
 	/**
-	 * enable chip when ready at a given strength
+	 * enable kick when ready at a given strength (0-255)
 	 */
-	void chip(uint8_t strength);
+	void kickLevel(unsigned char strength);
```

Comment:
Any reason for unsigned char over uint8_t?  It seems like int makes it a little clearer what it's for and it also makes it the same as the corresponding parameter to `chipLevel()`.